wsi/drm/drmdisplay.cpp

```diff
@@ -782,6 +786,20 @@ void DrmDisplay::SetPipeCanvasColor(uint16_t bpc, uint16_t red, uint16_t green,
                            canvas_color_prop_, canvas_color);
 }
 
+bool DrmDisplay::SetPipeMaxBpc(uint16_t max_bpc) const {
+  int ret;
+
+  if (max_bpc_prop_ == 0)
```

Comment:
Did you mean:
     if (max_bpc == 0)

Reply:
It is correct. He is checking for the existance of the property.
you could check `if (!max_bpc_prop_ || !max_bpc)`

Reply:
Okay, I thought @Kishore409 was checking the input parameter, max_bpc instead.

Everything else looks good.